File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int mul = 1;
        int sum = 0;
        for(int i = 0; i < num_list.length; i++) {
        	sum += num_list[i];
        	mul *= num_list[i];
        }
        return mul < Math.pow(sum, 2) ? 1 : 0;
    }
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    let answer = 1;
    let sum = 0;
    for(let i = 0; i < num_list.length; i++){
        answer *= num_list[i];
        sum += num_list[i];
    }
    
    return answer < Math.pow(sum, 2) ? 1 : 0;
}